extensions/cornerstone/src/types/CornerstoneServices.ts

```diff
@@ -0,0 +1,16 @@
+import { Types } from '@ohif/core';
+import ToolGroupService from './services/ToolGroupService';
+import SyncGroupService from './services/SyncGroupService';
+import SegmentationService from './services/SegmentationService';
+import CornerstoneCacheService from './services/CornerstoneCacheService';
+import CornerstoneViewportService from './services/ViewportService/CornerstoneViewportService';
+
+interface CornerstoneServices extends Types.Services {
```

Comment:
I don't think this is a good idea, we should be able to just extend the Types.Services and still be Types.Services everywhere, I don't like how each extension can just extend and claim it with their own name. Probably you can just do that inside the `registerServices` right?

Reply:
The problem is that there isn't any way to inject the type from the cornerstone extension into the @ohif/core component, so I don't really see how one could include the actual type there.  If you find a way, please let me know.